classes/class-wc-rest-connect-tos-controller.php

```diff
@@ -23,7 +23,7 @@ public function post( $request ) {
 		$settings = $request->get_json_params();
 
 		if ( ! $settings || ! isset( $settings[ 'accepted' ] ) || ! $settings[ 'accepted' ] ) {
-			return new WP_Error( 400 );
+			return new WP_Error( 'bad_request', __( 'Bad request' ), array( 'status' => 400 ) );
```

Comment:
Are these strings found in core WP, or are they missing the `woocommerce-services` domain?